electron/src/paths.ts

```diff
@@ -10,24 +10,51 @@ const DEV_USER_NUMBER = process.env.ACORN_AGENT_NUM
 
 const MIGRATION_FILE_NAME_PREFIX = `data-migration-for-integrity-version-`
 
-const USER_DATA_PATH = app.isPackaged
+const DEV_USER_DATA_PATH = app.isPackaged
```

Comment:
i was wrong about this being DEV
This is per environment. I was misled by lines below.

Reply:
Let’s remove DEV again, and instead fix the lower references to use this